package/features/environment.py

```diff
@@ -39,7 +39,7 @@
 from features.steps.util import create_new_venv
 
 
-def before_all(context):
+def before_scenario(context, feature):  # pylint: disable=unused-argument
```

Comment:
I moved all code in `before_all` into `before_scenario` (I know it's slowing down the CI), and the same for `after_all` to `after_scenario`, because Scenario 0.14.0 (or 0.14.3)  was making Scenario 0.15.0 and latest failing. In theory, each scenario should not interfere with each other, but `venv` was being shared across scenarios, which might be the reason why it was failing.

If anyone has a better idea, I'm happy to change this :)